word_language_model/main.py

```diff
@@ -243,7 +243,31 @@ def export_onnx(path, batch_size, seq_len):
 
 # Load the best saved model.
 with open(args.save, 'rb') as f:
-    model = torch.load(f)
+    if args.model == 'Transformer':
```

Comment:
I think I need to add more stuff to safe_globals. Spotted this by running example manually - it's likely the difference of `--dry-run` used in CI script. Working on the fix.

Reply:
Was not `--dry-run` actually, but different models like `Transformer` or `GRU`. Fixed by adding more stuff to safe globals and adding few more tests to CI script.